src/Components/Nav/Nav.js

```diff
@@ -42,8 +45,12 @@ export default class Nav extends Component {
     };
 
     render() {
+        const isEditMode = this.context && this.context.mode && this.context.mode === 'EDIT_MODE';
```

Comment:
You sure we need check each step? didn't `this.context` could be just a empty object.

Reply:
needed otherwise fails on SSR

Reply:
Sure? because [here](https://github.com/dotCMS/dotcms-spa-edit-mode/blob/28d95eaf6ffbec7d3ec57194ffb034512a64cdae/src/Components/Contentlet.js#L33) and [here](https://github.com/dotCMS/dotcms-spa-edit-mode/blob/2266f7f11de5bc4e9d21e9c6145cd2d54fd80791/src/Components/Container.js#L18) is not breaking in SSR.